gateway/README.md

```diff
@@ -105,9 +104,10 @@ func (s *myServiceImpl) MyMethod(ctx context.Context, req *MyRequest) (*MyRespon
 }
 ```
 
-If you do not use any custom outgoing header matcher you would see something like that:
+If you do not use any custom outgoing header matcher, you will see something like this.
+
 ```sh
-> curl -i http://localhost:8080/contacts/v1/contacts
+$ curl -i http://localhost:8080/contacts/v1/contacts
```

Comment:
Just a random thought. In contacts app we currently have `http://localhost:8080/v1/contacts`, but here it doesn't even matter for this example. I was thinking if it makes sense to change it to something neutral like `http://localhost:8080/data`. Up to you...